pkg/netceptor/conn.go

```diff
@@ -292,7 +304,11 @@ func (li *Listener) acceptLoop(ctx context.Context) {
 func (li *Listener) Accept() (net.Conn, error) {
 	select {
 	case ar := <-li.AcceptChan:
-		return ar.Conn, ar.Err
+		if ar == nil {
+			return nil, fmt.Errorf("listener closed")
```

Comment:
I suggested we differentiate this error message from the DoneChan error message.
Something like "listener accept channel closed"